### DISCUSSION TITLE
please assign me labels hacktoberfest-accepted on my pr for verified

### DISCUSSION THREAD

**Discussion by ajay-dhangar** [wingkwong/leetcode-the-hard-way#666] [General]
**pr :** https://github.com/wingkwong/leetcode-the-hard-way/pull/655

**Comment by wingkwong** [comment]
> YOUR PR/MRS MUST BE IN A REPO TAGGED WITH THE “HACKTOBERFEST” TOPIC, OR HAVE THE “HACKTOBERFEST-ACCEPTED” LABEL.

source: https://hacktoberfest.com/participation/

The repository has been tagged with `hacktoberfest`, so that I don't need to add the accepted label one by one. In short, you PR is counted. Please don't misuse discussion.

**Comment by ajay-dhangar** [comment]
Thanks, I viewed on my hacktoberfest profile.
